targets/arm.ld

```diff
@@ -69,3 +80,7 @@ _heap_start = _ebss;
 _heap_end = ORIGIN(RAM) + LENGTH(RAM);
 _globals_start = _sdata;
 _globals_end = _ebss;
+
+/* For the RAM persistence module. */
+_persist_start = _spersist;
+_persist_end = _epersist;
```

Comment:
Can you not just define these above in place of `_spersist` and `_epersist`? There doesn't appear to be anything else using these symbols, so I suggest getting rid of them.

Reply:
I was just aiming to follow the pattern in the file - i'm more than happy to just use _spersist, _epersist.  It would work fine.